instrumentation/lettuce/lettuce-5.1/testing/src/main/java/io/opentelemetry/instrumentation/lettuce/v5_1/AbstractLettuceSyncClientTest.java

```diff
@@ -135,6 +140,15 @@ void testSetCommand() {
                             .hasEventsSatisfyingExactly(
                                 event -> event.hasName("redis.encode.start"),
                                 event -> event.hasName("redis.encode.end"))));
+
+    List<AttributeKey<?>> expected =
+        new ArrayList<>(
+            Arrays.asList(
+                DB_SYSTEM, SERVER_ADDRESS, SERVER_PORT, NETWORK_PEER_ADDRESS, NETWORK_PEER_PORT));
```

Comment:
```suggestion
    List<AttributeKey<?>> expected =
        Arrays.asList(
            DB_SYSTEM, SERVER_ADDRESS, SERVER_PORT, NETWORK_PEER_ADDRESS, NETWORK_PEER_PORT);
```

Reply:
had to roll back - Arrays.asList is immutable - but we need to add an element later